tools/db/data/import.ts

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env -S npx ts-node
+
+/*
+		This script imports all the data from the `data` folder into a MongoDB database.
+		This is mainly used to bootstrap a database for e2es or to start developing on a clean database with just the minimal required data to run Kordis.
+
+		Usage: ./import.ts "<MONGO DB CONNECTION URI WITH DB>"
+ */
+import * as fs from 'fs';
+import { Collection, MongoClient } from 'mongodb';
+import * as path from 'path';
+
+import { CollectionData } from './collection-data.model';
+
+async function loadDataToMongoDB(connUri: string) {
+	try {
+		const client = await MongoClient.connect(connUri);
+
+		const db = client.db();
+
+		const filenames = fs
+			.readdirSync(__dirname)
+			.filter((filename) => filename.endsWith('.data.ts'));
+
+		await Promise.all(
+			filenames.map(async (filename) => {
+				const module = await import(path.join(__dirname, filename));
+				const dataModule: CollectionData = module.default;
```

Comment:
How about checking if dataModule is not `null` (`default export` could not be set) and checking that it conforms to the collection data model?

Reply:
I think that is not something we want to check at runtime. It will be too late for in corrupt structures, since we cannot change it. What about tests for the data files that validate everything and when someone checks in a new collection and messes something up, the tests will fail and not when it is actually used?

Reply:
Yes, agree with you. I intend to avoid that the import of a data file silently fails. 
I'm not sure about the test idea though. That might pollute our testing with non-application stuff 🤔

Reply:
may it is fine if we just keep it that way and check it during code review